OMCompiler/Compiler/SimCode/SimCodeVar.mo

```diff
@@ -104,6 +105,7 @@ public uniontype SimVar "Information about a variable in a Modelica model."
     Option<String> matrixName "if the varibale is a jacobian var, this is the corresponding matrix";
     Option<Variability> variability "FMI-2.0 variabilty attribute";
     Option<Initial> initial_ "FMI-2.0 initial attribute";
+    Option<Integer> fmi_variable_index "special variable index used for modelDescription.xml";
```

Comment:
Only variables with `fmi_variable_index != NONE()` will be exposed in the modelDescription.xml, right?

Reply:
this index is special set of index and it is updated only for the variables in modeldescription list 
list<SimVar> modelDescriptionVars